packages/components/src/PassengerCard/PassengerCard.js

```diff
@@ -0,0 +1,135 @@
+
+type Bag = {|
+  +count: number,
+  +type: string,
+|};
+
+type Props = {|
+  +blank: boolean,
+  +name: string,
+  +gender: 'female' | 'male' | 'other',
+  +nationality: string,
+  +dateOfBirth: string,
+  +id: string,
+  +insurance: string,
+  +bags: Array<Bag>,
+  +passengerCount: number,
+|};
+
+export default function PassengerCard(props: Props) {
+  const newPassengerTitle = `${props.passengerCount}. Passenger`;
+  const preffix =
+    props.gender === 'male' ? 'Mr.' : props.gender === 'female' ? 'Ms.' : '';
+  const passengerTitle = `${preffix} ${props.name}`;
+
+  const passenger = props.blank ? newPassengerTitle : passengerTitle;
+  const insurance = props.blank ? '' : props.insurance;
+  const dateOfBirth = props.blank ? '' : props.dateOfBirth;
+  const nationality = props.blank ? '' : props.nationality;
+  const id = props.blank ? '' : props.id;
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.container_name}>
+        <Icon name="passenger" />
+        <Text size="large">{passenger}</Text>
+      </View>
+      <View style={styles.container_top}>
+        <View style={styles.row_wrapper}>
+          <Text type="secondary" style={styles.text_padding}>
+            Nationality
+          </Text>
+          <Text>{nationality}</Text>
+        </View>
+        <View style={styles.row_wrapper}>
+          <Text type="secondary" style={styles.text_padding}>
+            Date of birth
+          </Text>
+          <Text>{dateOfBirth}</Text>
+        </View>
+        <View style={styles.id_row_wrapper}>
+          <Text type="secondary" style={styles.text_padding}>
+            ID
+          </Text>
+          <Text>{id}</Text>
+        </View>
+      </View>
+      <Separator />
+      <View style={styles.container_bottom}>
+        <View style={styles.row_wrapper}>
+          <Text type="secondary" style={styles.text_padding}>
+            Travel insurance
+          </Text>
+          <Text>{insurance}</Text>
+        </View>
+        <View style={styles.bags_row_wrapper}>
+          <Text type="secondary" style={styles.text_padding}>
+            Bags
+          </Text>
+          <View>
+            {props.blank ||
+              props.bags.map(bag => (
+                <BagInformation
+                  key={bag.type}
+                  count={bag.count}
+                  type={bag.type}
+                />
+              ))}
+          </View>
+        </View>
+      </View>
+    </View>
+  );
+}
+
+PassengerCard.defaultProps = {
+  passengerCount: 1,
+};
+
+const styles = StyleSheet.create({
+  container: {
+    paddingHorizontal: parseInt(defaultTokens.spaceMedium, 10),
+    borderColor: defaultTokens.borderColorCard,
+    borderTopWidth: 2,
+    borderBottomWidth: 2,
+  },
+  container_name: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    paddingTop: parseInt(defaultTokens.spaceSmall, 10),
+    paddingBottom: parseInt(defaultTokens.spaceMedium, 10),
+  },
+  container_top: {
```

Comment:
I'm not sure about snake case notation in styles. I think we should keep consistent style and use camelCase as everywhere

Reply:
There should be some eslint-rule that verifies camelCase 🤔

Reply:
Will change the case now, eslint rule would be great, didnt notice I was inconsistent in naming